src/extension/app/components/plugin/env-switcher/env-switcher.js

```diff
@@ -313,6 +313,11 @@ export class EnvironmentSwitcher extends ConnectedElement {
 
     this.appStore.switchEnv(value, openNewTab);
     picker.value = this.currentEnv;
+
+    this.appStore.sampleRUM('click', {
+      source: 'sidekick',
+      target: `switch-env:${value}`,
```

Comment:
should we stick with past tense, i.e. `envswitched`?

Reply:
Makes sense, will switch to `env-switched:${value}`